nemo_rl/models/policy/dtensor_policy_worker.py

```
@@ -1186,6 +1186,130 @@
+                    attention_mask = torch.zeros(
+                        (batch_size, seq_len), dtype=torch.long, device=input_ids.device
+                    )
+                    for i, length in enumerate(input_lengths):
+                        # For right-padded sequence, set 1s at the beginning of the sequence
+                        attention_mask[i, :length] = 1
+                    position_ids = torch.arange(
+                        seq_len, device=input_ids.device
+                    ).repeat(batch_size, 1)
+
+                with torch.autocast(device_type="cuda", dtype=self.dtype):
+                    attention_mask_input_all_ones = torch.ones(
```

Ruff (0.12.2)

Local variable `attention_mask_input_all_ones` is assigned to but never used

Remove assignment to unused variable `attention_mask_input_all_ones`

(F841)